client/main.lua

```diff
@@ -311,24 +307,116 @@ CreateThread(function()
         end
         if not inRange then
             Wait(1000)
-            if MainEntrance then
-                MainEntrance:remove()
-                MainEntrance = nil
+            if mainEntrance then
+                mainEntrance:remove()
+                mainEntrance = nil
             end
-            if MainExit then
-                MainExit:remove()
-                MainExit = nil
+            if mainExit then
+                mainExit:remove()
+                mainExit = nil
             end
         end
     end
 end)
 
-RegisterNetEvent('qb-newsjob:client:TakeOutVehicle', function(data)
-    local vehicle = data.vehicle
-    TakeOutVehicle(vehicle)
-end)
+CreateThread(function()
+    while true do
+        Wait(0)
+        local inRange = false
+
+        if IsLoggedIn then
+            local pos = GetEntityCoords(cache.ped)
+            if #(pos - vector3(config.locations.helicopterStorageEntrance.coords.xyz)) < 1.5 or #(pos - vector3(config.locations.helicopterStorageExit.coords.xyz)) < 1.5 then
+                inRange = true
+                if #(pos - vec3(config.locations.helicopterStorageEntrance.coords.xyz)) < 1.5 then
```

Comment:
Why are these distance checks needed? Seems like these zones should be created once; and not created and destroyed within a loop. Could clean them up on resource stop and create them on resource start though.

Reply:
The other comments are fine to leave for another PR, but this one has to do with code introduced here and should be addressed.